### ISSUE TITLE
[Screen Reader-Help-JavaScript-Call a function]: Role is not defined for all interactive elements on call a function pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2530]
**User Experience**
If any control does not have proper Role then the person with visual related disability will find it difficult to interpret the proper fields information. If role of component is not identified by screen reader then the person with vision disability will not be able to identify the control.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'JavaScript' control and select it.
5-Navigate to various controls lying on the pane opened.
6-Navigate to the pane opened and select any link listed under it.
7-Verify role is defined for all interactive controls or not on call a function pane.

**Actual Result**
Role is not defined for Below interactive elements on call a function pane.
1.Open documentation in a new tab
2.Edit 
3.Run 

**Expected Result**
Role should be defined for Below interactive elements on call a function pane.
1.Open documentation in a new tab
2.Edit 
3.Run 

**Note:** The same issue exists in scenario:
1. Help-->Projects-->Select any step like smiley buttons--> Role is not defined for List File code, Download and Play button

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested fix**
Using ARIA role to expose the control type of user interface component 
e.g. 
< div role="toolbar" tabindex="0" id="customToolbar" >  
< img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home" >

**Reference:** 
https://www.w3.org/TR/WCAG20-TECHS/aria.html 
https://www.w3.org/TR/WCAG20-TECHS/ARIA16.html  
https://www.w3.org/TR/WCAG20-TECHS/ARIA4.html  
https://www.w3.org/TR/WCAG20-TECHS/ARIA5.html 

**Please refer for attachment** 
![mas4 1 2_call a function_role is not defined for open a documentation](https://user-images.githubusercontent.com/30459830/28620173-77242a92-7229-11e7-843b-729ba2343ee5.png)
![mas4 1 2_call a function_role is not defined for run and](https://user-images.githubusercontent.com/30459830/28620174-7724c128-7229-11e7-8891-12620da11983.png)

**Comment by v-amik** [comment]
Review done

**Comment by veler** [comment]
The open documentation in new tab is now accessible [https://github.com/Microsoft/pxt/pull/2552](https://github.com/Microsoft/pxt/pull/2552). The other points are part of the documentation, I will work on it later, probably in a few days or weeks.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
